test/test_jit.py

```diff
@@ -5275,6 +5275,108 @@ def forward(self, input):
         m2.sub2.a.data.zero_()
         self.assertEqual(torch.zeros(2, 2), m2.forward(torch.randn(3, 2)))
 
+    def test_filecheck(self):
```

Comment:
This could move to its own class and all the inner functions become actual test cases since its not really doing any torchscript testing

Reply:
move to its own class or own file ? I thought it would be helpful to have some example usages in test_jit, since this is the only part of the library that will be testing conditions on strings (graphs)

Reply:
Yeah its own class